website/docs: release notes for 2026.5.0 (#21997)
goauthentik/authentik: website/docs/releases/2026/v2026.5.md

---
title: Release 2026.5
slug: "/releases/2026.5"
beta: true
---

## Highlights

- **Account Lockdown**: :ak-enterprise A "panic button" for Admins or account owners to immediately secure accounts suspected of being compromised by locking the user, deleting tokens, disabling passwords, and terminating sessions.
- **Fleet Conditional Access**: :ak-enterprise authentik can now verify user devices using Fleet certificates via the Fleet Connector and an mTLS stage, without the authentik agent.
- **AKQL is now open source**: The authentik query language for searching logs and users, previously enterprise-only, is now free for everyone to use.

> - **Command Palette and Wizard upgrades**: A new CMD + K command palette to search the authentik UI, alongside reworked wizards including a new user creation wizard, improved binding wizard, and new invitation wizard.

Reviewer: What's the w*ndows or linux equivalent? is it same keybinds

Reviewer: oh it's on L36, but would be worth mentioning here rq. Yk if you say one, you need to say the others

> - **Performance improvements**: A backend rework in Rust drops memory usage by approximately 200 MB per worker container, opens one fewer PostgreSQL connection per worker, and makes the Admin Interface less resource-intensive through lazy-loaded modals.

Reviewer: Ehhhhhh a bit much to call it a backend rework I say. It's just how the worker is started, I think? Threads, all that complex stuff

---
title: Release 2026.5
slug: "/releases/2026.5"
beta: true
---

## Highlights

- Account Lockdown :ak-enterprise

- Fleet Conditional Access / Chrome Device Trust :ak-enterprise

- AKQL Open-source

- Command Palette/Wizard upgrades

> - Rust stuff

Reviewer 2: ```suggestion
- Performance improvements
```

I think? Based on the title of the section below

## New features and improvements

### Account Lockdown: :ak-enterprise

> Account Lockdown allows you to immediately secure accounts that you suspect have been compromised. You can now do any combination of:

Reviewer: just putting this here as a reminder to update this

Author: Do you know when you will?

### Fleet Conditional Access: :ak-enterprise

authentik can now verify user devices based on their Fleet certificates without requiring the authentik agent, using the Fleet Connector together with an mTLS stage. For details, refer to the [Fleet Conditional Access documentation](../../endpoint-devices/device-compliance/fleet-conditional-access.md).

### WebAuthn Client Hints support

The WebAuthn Stage now supports the `hints` parameter from the [WebAuthn Level 3 spec](https://www.w3.org/TR/webauthn-3/#enum-hints). Admins can configure one or more hints (`security-key`, `client-device`, or `hybrid`) to tell the browser which authenticator type to expect. The browser uses this to skip straight to the relevant selection UI during passkey registration and authentication, reducing friction especially in enterprise deployments where security keys are mandatory.

Keep in mind that hints are advisory — they only affect the browser UI, not policy. Authenticator type requirements still need to be enforced server-side.

> 

Reviewer: ```suggestion

### TODO: Hashed passwords
```

### AKQL is now OSS

The authentik query language tool - AKQL - was an enterprise only feature that allowed for [querying logs](../../sys-mgmt/events/logging-events.mdx#advanced-queries) and [users](../../users-sources/user/user_basic_operations.md#advanced-queries). This allows you to search for events based on specific attributes, for example:

> context.geo.country = "Germany". We have open-sourced AKQL, meaning it is now free for everyone to use.

Reviewer: this should be on L52 as well, and within `backticks` (the akql query)

## New features and improvements

### Account Lockdown: :ak-enterprise

Account Lockdown allows you to immediately secure accounts that you suspect have been compromised. You can now do any combination of:

- defining a flow to lock a user
- deleting tokens
- disabling passwords
- terminating sessions

This acts as a "panic button" that can be pressed by an Admin or the owner of an account in case an account is thought to be compromised.

### General Improvements

### Command Palette

Command palette allows you to use CMD + K to search the authentik UI for different features and things you'd like to customize.

### Fleet Conditional Access: :ak-enterprise

authentik can now verify user devices based on their Fleet certificates, without the authentik agent. This is achieved with the Fleet Connector and an mTLS stage.

Link to docs: website/docs/endpoint-devices/device-compliance/fleet-conditional-access.md

### WebAuthn Client Hints support

> https://github.com/w3c/webauthn/blob/main/explainers/hints-for-webauthn-clients.md

Reviewer 3: Specs and other explainers, just in case we can improve our wording
https://www.w3.org/TR/webauthn-3/#enum-hints
https://developer.chrome.com/blog/passkeys-updates-chrome-129
https://passkeys.dev/docs/advanced/client-hints/

Author: I think we have it in a good spot now thanks to your help

> WebAuthn hints are an optional parameter that let you suggest to the browser which type of authenticator the user is likely to use, so the browser can present a more focused and relevant UI. For organization admins, this can promote users adopting the preferred authentication method and for users, this can help simplify the registration and login process.

Reviewer 3: ```suggestion
The WebAuthn Stage now supports the `hints` parameter from the WebAuthn Level 3 spec. Admins can configure one or more hints (`security-key`, `client-device`, or `hybrid`) to tell the browser which authenticator type to expect. The browser uses this to skip straight to the relevant selection UI during passkey registration and authentication, reducing friction especially in enterprise deployments where security keys are mandatory.

Keep in mind that hints are advisory — they only affect the browser UI, not policy. Authenticator type requirements still need to be enforced server-side.
```

context.geo.country = "Germany". We have open-sourced AKQL, meaning it is now free for everyone to use.

### Generally improved UI and accessibility

Wizards throughout authentik have been reworked, including a new user creation wizard, an improved binding wizard and a new invitation wizard. The invitation wizard in particular now makes it easy for administrators to send invites to new users. It guides admins through the process of configuring an invite system and sending the invites to users. Service accounts are now created in the same location as all other users. Every button in authentik now has a screen reader friendly label that tells the user the full action that will execute when the button is pressed. These changes have all been reflected in the docs.

**note** many accessibility changes that will be added by teffen

### Small general improvements (SAML Issuer, Hide apps)

SAML Issuer - We now automatically generate your SAML issuer url. You may still overwrite the default SAML Issuer.

> Hide Apps - You can hide apps from the Application Dashboard for situations when a user needs access to an application that doesn't need to show up in the Application Dashboard.

Reviewer 3: add link to the docs
https://next.goauthentik.io/add-secure-apps/applications/manage_apps/#hide-applications
and mention something about the info/warning, it's a change of behavior
`Before authentik 2026.5, an application was hidden by setting its Launch URL to blank://blank. Existing applications using that value are automatically migrated to using the Hide from My applications option upon upgrading.`

Author: resoved

### Performance improvements

> We've reworked a significant section of our backend in Rust. This had lead to an approximately 30% drop in memory usage. If you're a developer, check the updated Developer Docs to install Rust.

Reviewer 2: ```suggestion
We've started reworking a section of our backend code in Rust. So for, this has lead to an approximately 200 MB drop in memory usage for a single worker container. We plan on continuing making this rework to improve both resource consumption and performance. If you're a developer, check the updated Developer Docs to install Rust.
```

> There are also improvements to PostgreSQL connection handling, _more things that Marc and Jens can list_.

Reviewer 2: ```suggestion
With that rework, we also open one less PostgreSQL connection per worker, which should put less load on the database.
```
